packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx

```diff
@@ -0,0 +1,109 @@
+import {Component, State, h, Element, Prop} from '@stencil/core';
```

Comment:
[Diff (external link)](https://gist.github.com/btaillon/cab6ceec3b89e599c463fe749b52a93b/revisions#diff-3b0ecccb619f0cb9acc3ef0d88a903e4d5573507c9ffb6e4bdec1449016e34e7)

Reply:
Previous component was basically unused as it was not a finished product, could we just overwrite the original for this one?